k2/csrc/fsa_equivalent.cc

```diff
@@ -7,14 +7,119 @@
 #include "k2/csrc/fsa_equivalent.h"
 
 #include <algorithm>
+#include <queue>
 #include <random>
 #include <unordered_map>
 #include <unordered_set>
+#include <utility>
 #include <vector>
 
 #include "k2/csrc/fsa.h"
 #include "k2/csrc/fsa_algo.h"
 #include "k2/csrc/properties.h"
+#include "k2/csrc/util.h"
+#include "k2/csrc/weights.h"
+
+namespace {
+// out_weights[i] = weights[arc_map1[arc_map2[i]]]
+static void GetArcWeights(const float *weights,
```

Comment:
> When definitions in a .cc file do not need to be referenced outside that file, place them in an unnamed namespace or declare them static

Please refer to https://google.github.io/styleguide/cppguide.html#Unnamed_Namespaces_and_Static_Variables

So there is no need to add `static` here since they are already in
an anonymous namespace.

Reply:
Yes, you're right, both of them give internal linkage. Will remove, thanks!